metadata-ingestion-modules/airflow-plugin/tests/unit/test_airflow.py

```diff
@@ -317,7 +317,7 @@ def test_lineage_backend(mock_emit, inlets, outlets, capture_executions):
         assert all(map(lambda let: isinstance(let, Dataset), op2.outlets))
 
         # Check that the right things were emitted.
-        assert mock_emitter.emit.call_count == 19 if capture_executions else 11
+        assert mock_emitter.emit.call_count == 20 if capture_executions else 11
```

Comment:
@asikowitz we're modifyng this test since we have dataplatforminstance in the mock calls